README.md

```diff
@@ -44,22 +44,37 @@ geth --vm.evm=./libevmone.so
 ```
 
 ### Building from source
+To build the evmone EVMC module (shared library), test, and benchmark:
 
-To build the evmone EVMC module (shared library), test or benchmark.
-
-```bash
+1. Clone the repo and create a ```build``` directory:
+```
 git clone --recursive https://github.com/ethereum/evmone
 cd evmone
 mkdir build
 cd build
+```
 
+2. Build dependencies and (on Windows) generate a Visual Studio solution, then build the source:
+#### Linux
+```
 cmake .. -DEVMONE_TESTING=ON
 cmake --build . -- -j
+```
 
+#### Windows
+
+* **Visual Studio 2015**: ```cmake .. -DEVMONE_TESTING=ON -G "Visual Studio 14 2015 Win64"```
```

Comment:
I doubt it will build with VS 2015 as this projects is heavy C++17. Did you check this?